src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -18,7 +18,9 @@
         "Features": {
             "SupportsUI": false,
             "SupportsXml": true,
-            "SupportsAnonymizedExport": false
+            "SupportsAnonymizedExport": false,
+            "ProfileValidationOnCreate": false,
```

Comment:
>ProfileValidationOnCreate [](start = 13, length = 25)

Should these settings be part of `CoreFeatures` or `Features`? In the code they seem to be part of `CoreFeatureConfiguration`.

Reply:
I think CoreFeatureConfiguration came about because it was needed in a data layer, which can have a dependency on Core, but not any of the Shared.Core projects